content/guides/integrations/dropwizard.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Dropwizard"
+date: 2018-11-05T14:42:17-07:00
+draft: false
+class: "shadowed-image lightbox"
+aliases: [/integrations/dropwizard]
+weight: 5
+---
+
+- [Background](#background)
+- [How it works](#how-it-works)
```

Comment:
I think we can remove `[How it works]` as it causes an unnecessary indentation level.